### ISSUE TITLE
status not accessible to blind users using screen readers in RDPConf

### ISSUE THREAD

**Issue by a-singer** [stascorp/rdpwrap#419]
Hello,

Using RDPConf on Windows 7 Pro Sp1, I am unable to access the information as to the status of the listener and, more generally, any diagnostics information. I am totally blind and use the Jaws and NVDA screen readers to access Windows. I am able to access the controls in RDPConf and set them, but the only thing I can access in the program is the word "diagnostics". When a sighted person looks at the window, they can see the wrapper state is installed, the status of support, the listening status, and so on. It is not always easy or possible to have someone physically read the screen or a screenshot. Therefore, it would be helpful if the status information were available to screen readers as the controls currently are.
Thank you.
